tools/sigh.js

```diff
@@ -52,6 +44,10 @@ const steps = {
 };
 
 const eslintCache = '.eslint_sigh_cache';
+// Files to be deleted by clean, if they aren't in one of the cleanDirs.
+const cleanFiles = ['manifest-railroad.html', eslintCache];
+const cleanDirs = ['shell/build', 'shells/lib/build', 'build'];
+
```

Comment:
Might it be worthwhile considering using a config file?  Some project use package.json for this purpose.